### ISSUE TITLE
Papago translation is not working

### ISSUE THREAD

**Issue by Alienus772** [bbepis/XUnity.AutoTranslator#402]
Other translators work fine, but papago translator doesn't. This problem started a week ago.

**Comment by PantaFH** [comment]
Yes, I also encountered the same problem, it was still available before, probably last week or last month, because I am in China, I can only use **### papago** but it seems to be **dead**.

**Comment by I0lOl0I** [comment]
I can only use GoogleTranslateCompat, the other online translator services are all dead to me, they either just shut down or throw exceptions in the log and does nothing.
